extensions/blocks/map/edit.js

```diff
@@ -271,40 +343,65 @@ class MapEdit extends Component {
 				</Fragment>
 			</Placeholder>
 		);
+		// Only scroll to zoom when the block is selected, and there's 1 or less points.
+		const allowScrollToZoom = isSelected && points.length <= 1;
 		const placeholderAPIStateSuccess = (
 			<Fragment>
 				{ inspectorControls }
 				<div className={ className }>
-					<Map
-						ref={ this.mapRef }
-						scrollToZoom={ isSelected } // Only scroll to zoom when the block is selected.
-						mapStyle={ mapStyle }
-						mapDetails={ mapDetails }
-						points={ points }
-						zoom={ zoom }
-						mapCenter={ mapCenter }
-						markerColor={ markerColor }
-						onSetZoom={ value => {
-							setAttributes( { zoom: value } );
+					<ResizableBox
+						size={ {
+							height: mapHeight || 'auto',
+							width: '100%',
+						} }
+						showHandle={ isSelected }
```

Comment:
Just pointing out that this is a newer prop, not available in WP yet, but the component works perfectly fine without it (just, without the handle, obviously).

Reply:
Good thing to note. For reference, it was added in WordPress/gutenberg#18097. Prior to that, adding the `is-selected` class to the `ResizableBox` component was the way to make it appear, d9aab4dfa adds this as a fallback.